src/routes/+page.svelte

```diff
@@ -1,18 +1,9 @@
 <script lang="ts">
-	import Input from '$lib/components/forms/Input.svelte';
 	import Button from '$lib/components/Button.svelte';
-	import Search from '$lib/components/Search.svelte';
-	import Radio from '$lib/components/forms/Radio.svelte';
-	import Checkbox from '$lib/components/forms/Checkbox.svelte';
-	import Link from '$lib/components/forms/Link.svelte';
+	import Link from '$lib/components/Link.svelte';
 	import Icon from '$lib/components/Icon.svelte';
 
-	let search = '';
-	let value = '';
-	let number = 0;
-	let bool_1 = false;
-	let bool_2 = false;
-	let radio_value = '';
+	import { goto } from '$app/navigation';
```

Comment:
Just a detail, but i like my imports to have a specific order:
- Svelte stuff
- SvelteKit stuff
- Types
- Helpers
- General components
- Page specific components

This is just a proposal for this project, we can discuss it in class.